File: src/main/java/br/uff/tecnomarias/domain/entity/PessoaFisica.java
package br.uff.tecnomarias.domain.entity;

import javax.persistence.*;
import java.io.Serializable;

@Entity
public class PessoaFisica extends Pessoa implements Serializable {
    private static final long serialVersionUID = 1L;

    @OneToOne(cascade = CascadeType.ALL)
    @Column(name = "id_links")
    private Links links;

    public PessoaFisica() {
    }

    public Links getLinks() {
        return links;
    }

    public void setLinks(Links links) {
        this.links = links;
    }
}

File: src/models/PessoaFisica.js
import Pessoa from './Pessoa';

export class PessoaFisica extends Pessoa {

    constructor(pessoa, links) {
        super(pessoa);
        this._links = links;
    }

    get links() {
        return this._links;
    }

    set links(links) {
        this._links = links;
    }
}